app/views/admin/teachers/show.html.erb

```diff
@@ -23,4 +23,10 @@ end %>
   <%= govuk_link_to('View change history', admin_teacher_timeline_path(@teacher)) %>
 </p>
 
+<% if @teacher.induction_periods.none? %>
+  <p class="govuk-body">
+    <%= govuk_link_to("Add an induction period", new_admin_teacher_induction_period_path(@teacher)) %>
```

Comment:
We could update the view spec with two contexts, with and without `induction_period`, and get this covered there too.